internal/operators/mce/manifest.go

```diff
@@ -34,11 +37,16 @@ func Manifests() (openshiftManifests map[string][]byte, customManifests []byte,
 	return openshiftManifests, mceManifest, nil
 }
 
-func getSubscription() ([]byte, error) {
+func getSubscription(openshiftVersion string) ([]byte, error) {
+	mceVersion, err := getMCEVersion(openshiftVersion)
+	if err != nil {
+		return nil, err
+	}
+	mceChannel := fmt.Sprintf("stable-%s", swag.StringValue(mceVersion))
```

Comment:
in MCE this may apply, but the channel format is not standart this may break in the future 

here some examples
```
NAME                                          DISPLAY NAME                                            	DEFAULT CHANNEL
3scale-operator								Red Hat Integration - 3scale                             	threescale-2.13
advanced-cluster-management					Advanced Cluster Management for Kubernetes               	release-2.8
aci-containers-operator						Cisco ACI Containers Operator                            	stable
odf-operator								OpenShift Data Foundation  							   		stable-4.14
amq-broker-rhel8							Red Hat Integration - AMQ Broker for RHEL 8 (Multiarch)  	7.11.x
```

Reply:
But this is not generic though this is mce code, so its ok no ?

Reply:
it's OK - just an FYI